ShellPkg/Library/UefiShellDebug1CommandsLib/HexEdit/BufferImage.c

```diff
@@ -142,9 +142,13 @@ HBufferImageBackup (
   VOID
   )
 {
+  // MU_CHANGE [BEGIN]: Fix VS22 17.14 memcpy substitution
   HBufferImageBackupVar.MousePosition = HBufferImage.MousePosition;
 
   HBufferImageBackupVar.BufferPosition = HBufferImage.BufferPosition;
+  CopyMem (&HBufferImageBackupVar.MousePosition, &HBufferImage.MousePosition, sizeof (HBufferImage.MousePosition));
+  CopyMem (&HBufferImageBackupVar.BufferPosition, &HBufferImage.BufferPosition, sizeof (HBufferImage.BufferPosition));

```

Comment:
Did you mean to comment out the two previous lines to these CopyMems?

Reply:
Done